### ISSUE TITLE
does this still work? does it have token grabbers/ password grabbers inside? 

### ISSUE THREAD

**Issue by AstroOnYT** [DeadBread76/Raid-Toolbox#178]

**Comment by hidden-r4d** [comment]
yes it has 15 token grabbers and 139 password grabbers and it sends your ip to the nsa and also reports you to the fbi

**Comment by SabotageLuke** [comment]
> yes it has 15 token grabbers and 139 password grabbers and it sends your ip to the nsa and also reports you to the fbi

can confirm

**Comment by Dyno0311** [comment]
> Cap, I looked through the whole thing and there’s nothing. You’re safe, @AstroOnYT.

Nah, The Developer worked As an FBI Agent. He will send the users IP and information to the HQ

**Comment by hidden-r4d** [comment]
> > Cap, I looked through the whole thing and there’s nothing. You’re safe, @AstroOnYT.
> 
> Nah, The Developer worked As an FBI Agent. He will send the users IP and information to the HQ

Factual ^
